src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -185,74 +190,68 @@ def launch_fluent(
     start_transcript: bool = True,
     show_gui: bool = None,
 ) -> Session:
-    """Start Fluent locally in server mode or connect to a running Fluent
+    """Launch Fluent locally in server mode or connect to a running Fluent
     server instance.
 
     Parameters
     ----------
     version : str, optional
-        Selects either the ``"2d"`` or ``"3d"`` version of Fluent.
-        Default is ``"3d"``.
-
+        Dimensions for modeling. The default is ``None``, in which case ``"3d"``
+        is used. Options are ``"2d"`` and ``"3d"``.
     precision : str, optional
-        Selects either the ``"single"`` precision or ``"double"``
-        precision version of Fluent. Default is ``"double"`` precision.
-
+        Precision level. The default is ``None``, in which case ``"double"``
+        is used. Options are ``"single"``or ``"double"``.
     processor_count : int, optional
-        Specify number of processors. Default is 1.
-
+        Number of processors. The default is ``None``, in which case 1
+        is used.
     journal_filename : str, optional
-        Read the specified journal file.
-
+        Name of the journal file to read. The default is ``None``.
     meshing_mode : bool, optional
-        Launch Fluent in meshing mode
-
+        Whether to launch Fluent in meshing mode. The default is ``None``,
+        in which case Fluent is launched in meshing mode.
     start_timeout : int, optional
         Maximum allowable time in seconds to connect to the Fluent
-        server. Default is 100 seconds.
-
+        server. The default is ``100``.
```

Comment:
Did you intend to delete the units?

Reply:
@dnwillia-work I did intend to delete "seconds" from the default sentence as it is clearly stated in the description. The default in this case is only the value.